naughtystrings/naughtystrings.go

```diff
@@ -0,0 +1,37 @@
+// Package naughtystrings is a collection of strings that have a high probability of causing issues when used as user input.
+package naughtystrings
+
+import (
+	"encoding/json"
+	"os"
+	"path"
+	"runtime"
+)
+
+// Base64Encoded returns the strings encoded in base 64.
+func Base64Encoded() []string {
+	return strings("blns.base64.json")
+}
+
+// Unencoded returns the strings.
+func Unencoded() []string {
+	return strings("blns.json")
+}
+
+func strings(file string) []string {
```

Comment:
I know it's internal but `strings` name probably should be changed to something else to avoid confusion with Go `strings` package https://godoc.org/strings